example/quickstart/obcluster.yaml

```diff
@@ -9,9 +9,6 @@ spec:
   serviceAccount: "default"
   userSecrets:
     root: root-password
-    proxyro: proxyro-password
-    monitor: monitor-password
-    operator: operator-password
```

Comment:
also update observer's image

Reply:
Shall we update to the latest `4.2.1.3-103000032023122818`?